<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDlg</class>
 <widget class="QDialog" name="LoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FAQBrowser Login</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>User name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leUserName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lePassword">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btLogin">
       <property name="text">
        <string>Login</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btRegister">
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOptions">
       <property name="text">
        <string>Options</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btHelp">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="labelMessage">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
